package.json

```diff
@@ -47,6 +47,7 @@
     "bip66": "^1.1.5",
     "bitcoin-core": "^3.0.0",
     "bluebird": "^3.7.2",
+    "bolt11": "^1.3.2",
```

Comment:
which is the difference with parsePaymentRequest from "invoices"? if we will use this then maybe we must remove "invoices"

Reply:
Its a different library. I wasn't aware that we were already using `invoices`...